File: app/src/main/java/com/nicholasdoglio/eyebleach/data/model/Child.java
package com.nicholasdoglio.eyebleach.data.model;

import com.squareup.moshi.Json;

public class Child {
    @Json(name = "kind")
    private String kind;
    @Json(name = "data")
    private ChildData childData;

    public String getKind() {
        return kind;
    }

    public void setKind(String kind) {
        this.kind = kind;
    }

    public ChildData getChildData() {
        return childData;
    }

    public void setChildData(ChildData childData) {
        this.childData = childData;
    }
}

File: app/src/main/java/com/nicholasdoglio/eyebleach/model/Child.kt
package com.nicholasdoglio.eyebleach.model

import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName

class Child {

    @SerializedName("kind")
    @Expose
    var kind: String? = null
    @SerializedName("data")
    @Expose
    var data: Data_? = null

}
